src/cli/index.ts

```diff
@@ -0,0 +1,248 @@
+      return project;
+    }
+  );
+
+  switch (outputType) {
+    case "leopard": {
+      const leopard = await writeStep(`${chalk.bold("Converting")} project to ${chalk.white("Leopard")}.`, () => {
+        const leopard = project.toLeopard();
+        return leopard;
+      });
+
+      const fullOutputPath = resolve(process.cwd(), output);
+
+      await writeStep(`${chalk.bold("Exporting")} project to directory ${chalk.white(fullOutputPath)}.`, () => {
+        // First, make sure the output path is an empty directory (create it if it doesn't exist)
+        let existingFiles: string[];
+        try {
+          existingFiles = fs.readdirSync(fullOutputPath);
+        } catch (err) {
+          if (err instanceof Object && "code" in err) {
+            switch (err.code) {
+              case "ENOENT":
+                // Directory does not exist, create it
+                fs.mkdirSync(output, { recursive: true });
+                existingFiles = [];
+                break;
+              case "ENOTDIR":
+                throw new StepError("Output path is a file, not a directory.");
+              default:
+                throw err;
+            }
+          } else {
+            throw err;
+          }
+        }
+
+        if (existingFiles.length > 0) {
+          throw new StepError("Output directory is not empty.");
+        }
+
+        // Write the files
+        for (const [filename, contents] of Object.entries(leopard)) {
+          // Create directories as needed
+          fs.mkdirSync(resolve(fullOutputPath, filename, ".."), { recursive: true });
+
+          // Write the file
+          fs.writeFileSync(resolve(fullOutputPath, filename), contents);
+        }
+
+        for (const target of [project.stage, ...project.sprites]) {
+          for (const costume of target.costumes) {
+            const filename = `${target.name}/costumes/${costume.name}.${costume.ext}`;
+            const asset = Buffer.from(costume.asset as ArrayBuffer);
+            fs.mkdirSync(resolve(fullOutputPath, filename, ".."), { recursive: true });
+            fs.writeFileSync(resolve(fullOutputPath, filename), asset);
+          }
+
+          for (const sound of target.sounds) {
+            const filename = `${target.name}/sounds/${sound.name}.${sound.ext}`;
+            const asset = Buffer.from(sound.asset as ArrayBuffer);
+            fs.mkdirSync(resolve(fullOutputPath, filename, ".."), { recursive: true });
+            fs.writeFileSync(resolve(fullOutputPath, filename), asset);
+          }
+        }
+      });
+
+      process.stdout.write(chalk.green`\nProject exported to Leopard format.`);
+      process.stdout.write(chalk.gray` Files written to ${chalk.white(fullOutputPath)}\n\n`);
+      process.stdout.write(
+        chalkBox([
+          { str: "To preview the project, run:" },
+          { str: `$ ${chalk.white`cd ${output}`}`, length: 5 + output.length },
+          { str: `$ ${chalk.white`npx vite`}`, length: 10 }
```

Comment:
On a principles level I don't really know about suggesting one arbitrary simple HTTP server over another, but obviously it's nice to have something here, and `npx vite` should generally on the system where the file was generated, at least. You could have some text like "or serve with any HTTP server", but if you don't think it's necessary, that's OK.